experimental/packages/otlp-exporter-base/src/platform/browser/OTLPExporterBrowserBase.ts

```diff
@@ -33,7 +33,7 @@ export abstract class OTLPExporterBrowserBase<
   ExportItem,
   ServiceRequest
   > {
-  protected _headers: Record<string, string>;
+  public _headers: Record<string, string>;
```

Comment:
This is because this class is passed as a parameter to an independent [send](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/otlp-proto-exporter-base/src/util.ts#L43) function, which does the actual export request, and this send function needs access to url, timeoutMillis and headers in OTLPProtoExporterNodeBase/OTLPProtoExporterBrowserBase. The [url and timeoutMillis](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/otlp-exporter-base/src/OTLPExporterBase.ts#L34) are already public, hence making headers public too.